tests/testthat/test-fixed_regex_linter.R

```diff
@@ -270,19 +270,6 @@ test_that("fixed replacement is correct with UTF-8", {
   )
 })
 
-# TODO(michaelchirico): one difference for stringr functions vs. base is that
```

Comment:
This is done already, I think on the last release when we did pipe-awareness consistency.